packages/helix_editor.rb

```diff
@@ -1,52 +1,62 @@
 require 'package'
 
-class Helix_editor < Package                 # The first character of the class name must be upper case
+class Helix_editor < Package
   description 'A vim inspired editor with LSP support'
   homepage 'https://helix-editor.com/'
   version '22.12'
   license 'MPL-2.0' # license of source
   compatibility 'all'
-  source_url 'https://github.com/helix-editor/helix/archive/refs/tags/22.12.tar.gz'
-  source_sha256 'edae8af46401b45c3e71c38b4fa99f931c4458127978ccd1b29aaae79331d972'   # Use the command "sha256sum"
-  
+  source_url 'https://t.ly/-vk7'
+  source_sha256 'edae8af46401b45c3e71c38b4fa99f931c4458127978ccd1b29aaae79331d972' # Use the command "sha256sum"
+  source_url 'file:///usr/local/home/helix/deleteme.tar.gz'
+  source_sha256 'c3a0e022a7bcd6b139ae7abf1e0dae33c8f9ec82dc680076d9e5ea6593b85bc6'
+
   depends_on 'rust' => :build
+  depends_on 'xdg_base'
 
   @no_fhs = true
-  
+
+  @xdg_config_home = ENV.fetch 'XDG_CONFIG_HOME', '/usr/local/.config'
+  @helix_runtime_dir = "#{@xdg_config_home}/helix"
+
   def self.build
-   puts "Building. This may be long."
-   system "cargo build \
+    puts 'Building. This may be long.'
+    puts 'cargo build \
       --release \
       --locked \
-     "
+     '
+  end
+
+  def self.check
+    # This will raise an error if "hx" is not a valid command
+    `hx --version`
```

Comment:
At this stage, it will still be located at `./target/release/hx`, correct?

Reply:
At this stage, it seems to me that the files are located as if the installation was finished. So /usr/local/bin/hx should be valid I think.
I don't know if ./target/release/hx is still valid; I haven't tried to access it.

Reply:
Or maybe I am wrong and hx was already installed when I self.check was executed? ^^

Reply:
You are right it is at `./target/release/hx` (what I said previously doesn't make sense).
I will only try './target/release/hx --version' then.

Would it be right to move the current code inside postinstall? I wanted to check the installation was working properly...

Reply:
Fixed self.check. I moved the old code in postinstall to print warning in case the installation doesn't work as expected.